data/hai/unfettered 2 war preparations.txt

```diff
@@ -0,0 +1,2164 @@
+			choice
+				`	"What kind of materials do you need?"`
+			`	Whether she hears your question or not, she responds to it explicitly, speaking to all that are present, "Do not bother with silicon, friends, we already have plenty, and its uses are limited. No, we need a lot of silver and lead. Also do not forget we are <selling> asteroid scanners in the outfitter, should you require any.`
+			`	"Titanium would also be <welcomed>, but as you well know there is only a very limited amount of it left in this system, and the fact there is a lot in Ik'kara'ka, as the aliens of the north call it, does little to help us."`
+			label speakover
+			`	It looks like Mera is done speaking to the others, and she comes to you, "Hello, <first>. It's been a while since I last saw you. Heard you took care of that Auxiliary and brought back quite the treasure; great work!" Her tail moves from left to right as she says this, which is probably a sign of excitement. "The renovation efforts are progressing at rapid pace, but we still need all the hands we can get!"`
+			`	You see how she looks at the translator you have and give it back before doing anything else.`
+				to display
+					has "unfettered: used translator"
+			label choice
+			choice
+				`	"I'll go collect the minerals, then."`
+					accept
+				`	"How should I collect minerals from asteroids?"`
+					to display
+						not "unfettered: asteroid tuto"
+				`	"From which systems can I get those resources from?"`
+					to display
+						not "unfettered: asteroid systems"
+					goto where
+				`	"Mining is not my thing. You'll have to do this one without me."`
+					goto nobiggie
+			label asteroidtuto
+			action
+				set "unfettered: asteroid tuto"
+			`	She seems a bit surprised, "You've never mined asteroids? I recommend you install an asteroid scanner so your ship can target them automatically. Just try to keep your ship stable around one and shoot it as you would any other ship. Keep in mind they have no shields, and you might not want to waste missiles on them, since they don't shoot back. If you ever run out of targets, you can fly around for a couple of days and come back; new asteroids will likely enter the system in that time."`
+				goto choice
+			label where
+			action
+				set "unfettered: asteroid systems"
+			`	Lead can simply be found right over our heads, and there are silver asteroids present in Ehma Ti. The lead asteroid looks similar to copper but is bigger. You can also just differentiate them with an asteroid scanner. Silver is easy to recognize with its very small size and natural sturdiness. As for titanium, I know there is some in Ik'kara'ka, up North, I can point the location on your map if you want."`
+				goto choice
+			label nobiggie
+			`	"Oh, I see." Her tail movement stops at once. "Are you sure? This would mean a lot for me, a lot of Unfettered refuse to collect resources this way already, preferring to shoot enemies that fire back."`
```

Comment:
```suggestion
			`	"Oh, I see." Her tail movement stops at once. "Are you sure? This would mean a lot for me, because a lot of Unfettered refuse to collect resources this way, instead preferring to attack enemies that shoot back."`
```